external/anomaly/anomaly_classification/configs/stfpm/template.yaml

```diff
@@ -18,6 +18,12 @@ entrypoints:
 # Hyper Parameters
 hyper_parameters:
   base_path: ./configuration.yaml
+  parameter_overrides:
```

Comment:
But if these values are the same for 'padim' and 'stfpm' and as far as I can see they are also equal to default configuration, are not they? Why do add them here?

Reply:
for padim the same question

Reply:
done